src/test/java/com/shinesolutions/aemorchestrator/service/AemInstanceHelperServiceTest.java

```diff
@@ -209,8 +205,6 @@ public void testGetPublishIdToSnapshotFromWithSameLaunchTime() throws Exception
         when(awsHelperService.getLaunchTime(alphabeticallyFirst)).thenReturn(originalDate);
         when(awsHelperService.getLaunchTime(alphabeticallySecond)).thenReturn(originalDate);
 
-        when(httpUtil.isHttpGetResponseOk(anyString())).thenReturn(true);
```

Comment:
Unnecessary stubbing `httpUtil.isHttpGetResponseOk`, is not invoked when `aemHelperService.getPublishIdToSnapshotFrom` is called.